setup.py

```diff
@@ -99,6 +108,7 @@ def finalize_options(self):
     package_data={
         'onnxruntime': data + examples + extra,
     },
+    py_modules=python_modules_list,
```

Comment:
is this the right use of py_modules ?

Reply:
We're exposing the python modules at the root level. py_modules seems to be the appropriate option to use for such cases as described in the[ python documentation](https://docs.python.org/3.5/distutils/setupscript.html#listing-individual-modules).

Reply:
i think py_modules is if you want to list all the modules in this python package individually.
i don't think it makes sense to use both packages and py_modules at the same time.
in any case, openvino_mo and openvino_emitter should only required in an openvino enabled package, and not as default modules.

Reply:
Done. Used py_modules, but the code now only adds the openvino_mo and openvino_emitter to it if the build options has --use_openvino argument.